src/node/executor.cpp

```diff
@@ -175,7 +212,7 @@ void Executor::execute_(PbftBlock const &pbft_block) {
     ws_server_->newEthBlock(new_eth_header);
   }
 
-  transactions_tmp_buf_.clear();
+  clearTmpContainers();
```

Comment:
you created this function just for the sake of giving a name to a block of code? seems excessive to me

Reply:
no i created it to clear all containers that need to be cleared and have it in one place. seems much better to me

Reply:
but yeah if i just put clear method for all thoise containers here it would the the same effect, I just found it better to have it grouped in function... technicality...

Reply:
I'm fine with it, but I wouldn't have written it like this myself :)